Command `wb_channel set` fails to send reply on success

When using the `wb_channel set` command, the response from the Discord client is "The application failed to respond".

Full stack trace:
```
Error [InteractionNotReplied]: The reply to this interaction has not been sent or deferred.
    at ChatInputCommandInteraction.followUp (..../weatherbot/node_modules/discord.js/src/structures/interfaces/InteractionResponses.js:177:64)
    at ..../weatherbot/src/commands/season.ts:39:23
    at Generator.next (<anonymous>)
    at ..../weatherbot/node_modules/tslib/tslib.js:118:75
    at new Promise (<anonymous>)
    at Object.__awaiter (..../weatherbot/node_modules/tslib/tslib.js:114:16)
    at Object.run (..../weatherbot/src/commands/season.ts:25:155)
    at ..../weatherbot/src/listeners/interactionCreate.ts:26:22
    at Generator.next (<anonymous>)
    at ..../weatherbot/node_modules/tslib/tslib.js:118:75
[ERROR] 13:23:54 Error [InteractionNotReplied]: The reply to this interaction has not been sent or deferred.
```

--- a/src/commands/season.ts
+++ b/src/commands/season.ts
@@ -36,6 +36,6 @@ export const Season: Command = {
     BotConfig.getInstance().save(config);
     log.info(content);
 
-    await interaction.followUp({ ephemeral: true, content });
+    await interaction.reply({ ephemeral: true, content });
   },
 };

